src/rest/encounterRest.js

```diff
@@ -43,6 +43,15 @@ const api = {
         return response.data;
       }
     }),
+    
+  fetchEncountersByObs: (patient, concept) => axiosInstance.get(`encounter?s=byObs&patient=${patient}&obsConcept=${concept}&v=custom:(uuid,obs)`)
```

Comment:
Let's change this so instead of returning the custom (uuid,obs) response it returns the default encounter rep, like the other endpoints in this file.

However, let's change all the methods into encounterRest to take a new optional parameter "representation", that, if specified, is used instead of the "default encounter rep".